src/core_modules/capture-core/components/DataEntry/withCleanUp.js

```diff
@@ -16,6 +16,6 @@ const getCleanUpHOC = (InnerComponent: React.ComponentType<any>) =>
         );
     };
 
-export default () =>
+export const withCleanUpHOC = () =>
```

Comment:
Hey, 
Do you think is necessary to add the `HOC` suffix to this component? There are other HOCs in this PR that follow a different naming pattern like `withCancelButton` or `withBrowserBackWarning`. I would suggest settling on a convention and apply it in all cases. What do you think?

Reply:
I think you're right, the name should be changed to withCleanUp.